src/services/walkthroughProvisionServices.js

```diff
@@ -0,0 +1,56 @@
+import {
+  buildValidProjectNamespaceName,
+  findOpenshiftResource,
+  findOrCreateOpenshiftResource,
+  namespaceRequestDef,
+  namespaceRequestResource,
+  namespaceDef,
+  namespaceResource,
+  serviceInstanceDef
+} from '../common/openshiftHelpers';
+import { currentUser, create } from './openshiftServices';
+import { DEFAULT_SERVICES, buildServiceInstanceResourceObj } from '../common/serviceInstanceHelpers';
+
+/**
+ * Provisions the services required for Walkthrough 1 into a username prefixed namespace.
+ * @param {Object} amqCredentials The credentials from the AMQ instance, including URL.
+ */
+const provisionWalkthroughOne = amqCredentials =>
+  currentUser().then(user => {
+    const namespace = buildValidProjectNamespaceName(user.username, 'walkthrough-one');
+    const siDef = serviceInstanceDef(namespace);
+    return findOpenshiftResource(namespaceDef, namespaceResource(namespace))
```

Comment:
Can you use `findOrCreateOpenshiftResource` here?

Reply:
Sadly not, it's finding a `Project`, but creating a `ProjectRequest`